public/styles/thumbnail.css

```
@@ -0,0 +1,83 @@
+
+.skeleton-rate {
+  height: 12px;
+  border-radius: 4px;
+  background: linear-gradient(90deg, #2a2a2a 25%, #3a3a3a 50%, #2a2a2a 75%);
+  background-size: 200% 100%;
+  animation: shimmer 1.4s infinite;
+  margin-bottom: 4px;
+  width: 40%;
+}
+
+.item.skeleton .item-desc > *:not(.skeleton-rate):not(.skeleton-title) {
```

Stylelint (17.5.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)